src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,20 @@
+package calculator;
+
+import org.springframework.util.StringUtils;
+
+import java.util.Arrays;
+
+public class StringCalculator {
+
+    public int add(String text) {
+        if (StringUtils.isEmpty(text)) {
```

Comment:
`org.springframework.util` 하위의 StringUtils도 좋지만, 
Java8에서 기본적으로 제공되어지는 `Strings.isBlank`를 사용하는건 어떨까요?

```suggestion
        if (Strings.isBlank(text)) {
```

Reply:
네, 의미없는 공백을 제외시킬 수 있어서 더 좋네요. 😍